Source/JavaScriptCore/builtins/PromiseConstructor.js

```diff
@@ -78,6 +79,106 @@ function all(iterable)
     return promiseCapability.@promise;
 }
 
+@linkTimeConstant
+function promiseOnRejectedWithContext(argument, context)
```

Comment:
Could we name this `promiseAllOnRejected` for consistency? It looks quite generic at a glance, but in actuality `context.globalContext.promise` is unlikely to occur elsewhere, right?